test/mlir/driver/version.mlir

```diff
@@ -0,0 +1,4 @@
+// RUN: onnx-mlir --version | FileCheck %s
+// XFAIL: onnx-mlir-vendor
+
+// CHECK: onnx-mlir version 0.3.0 ({{.*}}/onnx-mlir{{.*}} {{[a-zA-Z0-9]+}}, {{.*}}/llvm-project{{.*}} {{[a-zA-Z0-9]+}})
```

Comment:
Don't hardcode the version here please, it will require the test to be updated every time the version changes.

Reply:
Done.